app/src/main/java/zapsolutions/zap/channelManagement/ChannelListItem.java

```diff
@@ -14,4 +16,104 @@
     abstract public int getType();
 
     abstract public ByteString getChannelByteString();
+
+    @Override
+    public int compareTo(ChannelListItem channelListItem) {
+        ChannelListItem other = channelListItem;
+
+        Long ownCapacity = 0l;
+        switch (this.getType()) {
+            case TYPE_OPEN_CHANNEL:
+                ownCapacity = ((OpenChannelItem) this).getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_OPEN_CHANNEL:
+                ownCapacity = ((PendingOpenChannelItem) this).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_CLOSING_CHANNEL:
+                ownCapacity = ((PendingClosingChannelItem) this).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_FORCE_CLOSING_CHANNEL:
+                ownCapacity = ((PendingForceClosingChannelItem) this).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_WAITING_CLOSE_CHANNEL:
+                ownCapacity = ((WaitingCloseChannelItem) this).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_CLOSED_CHANNEL:
+                ownCapacity = 0l;
+        }
+
+        Long otherCapacity = 0l;
+        switch (other.getType()) {
+            case TYPE_OPEN_CHANNEL:
+                otherCapacity = ((OpenChannelItem) other).getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_OPEN_CHANNEL:
+                otherCapacity = ((PendingOpenChannelItem) other).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_CLOSING_CHANNEL:
+                otherCapacity = ((PendingClosingChannelItem) other).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_PENDING_FORCE_CLOSING_CHANNEL:
+                otherCapacity = ((PendingForceClosingChannelItem) other).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_WAITING_CLOSE_CHANNEL:
+                otherCapacity = ((WaitingCloseChannelItem) other).getChannel().getChannel().getCapacity();
+                break;
+            case TYPE_CLOSED_CHANNEL:
+                otherCapacity = 0l;
+        }
+
+        return ownCapacity.compareTo(otherCapacity);
```

Comment:
Do you think its clear for a user how the sorting is done?
I had to look twice to get it, as it seemed pretty random on first glance. (Not by alias, not by send capacity, not receive capacity - so the visual representation is kinda disassociated from the sorting)

Reply:
Yes, that is true. Before it was sorted by opening date, which is also not visible.
Would you prefer sorting by alias? Or what would you suggest?

Reply:
I ordered it by alias now, and I like it much better.